pkg/ipmi/ipmi.go

```diff
@@ -377,3 +327,7 @@ func (i *IPMI) RawCmd(param []byte) ([]byte, error) {
 
 	return i.RawSendRecv(msg)
 }
+
+func (i *IPMI) Close() error {
+	return i.Handler.GetFile().Close()
```

Comment:
Would prefer if Handler had a Close() method rather than a GetFile() on which Close() can be called. That way, an implementer of Handler cannot accidentally return a different *os.File object in GetFile() and leave something unclosed.